backend/app/helpers/spree/admin/orders_helper.rb

```diff
@@ -6,7 +6,8 @@ def event_links
         links = []
         @order_events.sort.each do |event|
           next unless @order.send("can_#{event}?")
-          links << button_link_to(Spree.t(event), [event, :admin, @order],
+          links << button_link_to(Spree.t(event).capitalize,
```

Comment:
Probably don't want to do it like this. The choice to capitalize or not is language specific, so it should exist somewhere in translations.